third_party/2and3/pycurl.pyi

```diff
@@ -21,8 +21,12 @@ class error(Exception): ...
 class Curl(object):
     def close(self) -> None: ...
     def setopt(self, option: int, value: Any) -> None: ...
+    def setopt_string(self, option: int, value: str) -> None: ...
     def perform(self) -> None: ...
+    def perform_rb(self) -> ByteString: ...
```

Comment:
Probably this returns a concrete `bytes` object? I wasn't able to install the library to make sure, but the source at https://github.com/pycurl/pycurl/blob/cd0e9cfcf63174a5dca8400a2e47d5ecebb26c4a/src/easyperform.c#L27 suggests it returns the return value of the `getvalue()` method on a `BytesIO` object.

Reply:
Yes, it will return bytes as per http://pycurl.io/docs/latest/curlobject.html#perform-rb .
It’s my personal preference to use `ByteString` instead of `bytes`. The later is just a shortened of former. It’s OK to use one that meets the project convention. :)

Reply:
Actually ByteString isn't the same as bytes; it's an abstract class. There's no need to use it here.

Reply:
(Made the change myself, I'll merge once CI passes.)